### PR TITLE
feat(docker): py3

### PR DESCRIPTION
**Pull request by scintillating7**
### Deployment changes
- Updated docker container image to run on python3.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
     gcc \
- && pip install wheel \
- && pip install -r build/requirements.txt \
- && python setup.py install 
+ && pip3 install wheel \
+ && pip3 install -r build/requirements.txt \
+ && python3 setup.py install
 
-FROM quay.io/ncigdc/apache-base:2.4.18-1.0.0
+FROM quay.io/ncigdc/apache-base:python3
```
Comment (ProfOak) [inline Dockerfile]: Is this the final tag going forward, or should we wait on merging this until it receives a numbered tag?

Comment (singergr) [reply Dockerfile]: Yeah, we should make an apache-base:1.0.2-python3 or something

Comment (singergr) [reply Dockerfile]: Tag will be `1.0.2-py3` once https://github.com/NCI-GDC/apache-base/pull/8 is merged and tagged.

**Comment by huarngpa** [comment]
> ### Deployment changes
> * Updated docker container image to run on python3.

I think we should close this one in favor of [TT-1138](https://github.com/NCI-GDC/indexd/pull/14). 

Reasons,
* Is it really python3? I don't think this works with anything 3.5+ (needs psycopg2-binary)
* Circular dependency between indexclient and indexd, TT-1138 changes fix this
* Rest of the dependency breaking with CDIS relies on TT-1138, see [here](https://github.com/NCI-GDC/gdcapi/pull/1237)

File: Dockerfile (hunk 1 above)
Comment (ProfOak) [reply Dockerfile]: I know we changed the top FROM directive, but do we need to change this one as well?

**Comment by singergr** [comment]
> Since we are running these on a container, can we target a higher version of Python instead of 3.5

Let's get everything running on 3.5 first and then look into going further.

File: Dockerfile (hunk 1 above)
Comment (singergr) [reply Dockerfile]: > I know we changed the top FROM directive, but do we need to change this one as well?

Yep, the bottom `FROM` directive is what goes into the final image.

**Comment by singergr** [comment]
Also, I should've mentioned this in my above comment, but we just released `1.0.3-py3.5`. :eyes:

**Pull request merged**